modules/configuration-ovnk-network-plugin-json-object.adoc

```diff
@@ -37,7 +41,9 @@ For `"topology":"layer2"` deployments, IPv6 (`2001:DBB::/64`) and dual-stack (`1
 
 |`netAttachDefName`
 |`string`
-| The namespace and the network attachment name. The value must match the values specified for `namespace` and `name` when defining the network attachment definition object. For example, `ns1/l2-network` refers to the `ns1` namespace and the `l2-network` network attachment definition name.
+|The metadata `namespace` and `name` of the network attachment definition object where this
+configuration is included. For example, if this configuration is defined in a network attachment
+definition in namespace `ns1` named `l2-network`, this should be set to `ns1/l2-network`.
```

Comment:
nit: should we use `'NetworkAttachmentDefinition'` instead of `network attachment
definition` as elsewhere?